package.json

```
@@ -21,7 +21,8 @@
     "*.{js,ts,vue}": [
       "eslint --fix",
       "git add"
-    ]
+    ],
+    "*.{css,vue}": "stylelint"
```

assets/ ディレクトリに scss がありました

```suggestion
    "*.{css,scss,vue}": "stylelint"
```